src/mescal/util.clj

```diff
@@ -55,3 +57,12 @@
 (defn get-enum-values
   [value-obj]
   (find-value value-obj [:enumValues :enum_values]))
+
+(defn encode-path
+  "Encodes a file path for use in a URL."
+  [root-dir file-path]
+  (as-> file-path path
```

Comment:
✨

Reply:
I think that `as->` is my new favorite macro! :)